src/components/new-safe/create/steps/StatusStep/useSafeCreation.ts

```diff
@@ -59,7 +62,7 @@ export const useSafeCreation = (
   )
 
   const handleCreateSafe = useCallback(async () => {
-    if (!pendingSafe || !provider || !chain || !wallet || isCreating) return
+    if (!pendingSafe || !provider || !chain || !wallet || isCreating || !maxFeePerGas) return
```

Comment:
Do we need this early return? This would prevent safe creation if there is no `maxFeePerGas`. Safe creation worked without it before for all wallets except Ledger afaik.